### ISSUE TITLE
Provide mapping config for more Commands of the built in Plugins

### ISSUE THREAD

**Issue by primeapple** [NvChad/NvChad#915]
**Is your feature request related to a problem? Please describe.**
I would like to change a mapping for a built-in plugin, lets say the `:Telescope treesitter` one.
Currently this requires me to add a custom mapping like
```lua
   map("n", "ft", ":Telescope treesitter <CR>")
```
in my own mappings file. This seems weird, because I can configure other mappings of telescope directly through the `chadrc.lua`

**Describe the solution you'd like**
Provide an interface for  the most important commands of all the built in plugins. Set unused mappings to `nil`.  
The important files should be `lua/core/mappings.lua`

If you want I can add a PR to start with Telescope, as well as Nvim-Tree.

**Comment by siduck** [comment]
you can directly configure through other mappings of telescope as they're just some default mappings and mentioned in mappings.lua where as telescope treesitter mapping isnt mentioned in mapping.lua hence you must use the map function for it and define it

**Comment by primeapple** [comment]
Yes, I know, however currently we have only a handful of mappings in `mapping.lua`.  
This makes me split my Telescope config into 2 files, my `chadrc.lua` as well as my `init.lua`, which is probably not the desired behaviour.

So the propose would be to add a couple more often used mappings for the inbuilt plugins to `mapping.lua`

**Comment by siduck** [comment]
> Yes, I know, however currently we have only a handful of mappings in `mapping.lua`. This makes me split my Telescope config into 2 files, my `chadrc.lua` as well as my `init.lua`, which is probably not the desired behaviour.
> 
> So the propose would be to add a couple more often used mappings for the inbuilt plugins to `mapping.lua`

telescope has tons of commands and it'd be messy to add them all, Also you just need to add the mapping in custom/init.lua. Not in chadrc

**Comment by primeapple** [comment]
I see.
My problem is that I would like to have all my mappings for a plugin in one place. Currently the ones for Telescope, Nvim-Tree, etc. are split between overwrites for NvChad mappings (which are done in `chadrc.lua`) and my own mappings (`init.lua`).

I thought at least adding some sane defaults for the most commonly used ones  could be added to `mapping.lua`
